<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZegoImageShowDialog</class>
 <widget class="QDialog" name="ZegoImageShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="m_zoneTitle" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #0e88eb;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="m_lbRoomName">
          <property name="styleSheet">
           <string notr="true">padding-left: 20px;
font-family: 微软雅黑;
font-size: 14px;
color: #ffffff;
border: none;</string>
          </property>
          <property name="text">
           <string>截图结果</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ZegoImageButton" name="m_bMin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>40</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    border: 1px solid #0e88eb;
    background-color: #0e88eb;
}

QPushButton:hover{
    border: 1px solid #0d80de;
    background-color: #0d80de;
}

QPushButton:hover:pressed{
    border: 1px solid #0d79d1;
    background-color: #0d79d1;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ZegoImageButton" name="m_bClose">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>40</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
    border: 1px solid #0e88eb;
    background-color: #0e88eb;
}

QPushButton:hover{
    border: 1px solid #0d80de;
    background-color: #0d80de;
}

QPushButton:hover:pressed{
    border: 1px solid #0d79d1;
    background-color: #0d79d1;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_ImageShowLabel">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>360</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>960</width>
         <height>540</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #333333;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_bQuit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!enabled{
     font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #e6e6e6;
     border-radius: 2px;
     background-color: #ffffff;
     color: #e6e6e6;   
}
QPushButton:enabled:!hover{
     font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #0e88eb;
     border-radius: 2px;
     background-color: #ffffff;
     color: #0e88eb;   
}

QPushButton:enabled:hover{
    font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #0e88eb;
     border-radius: 2px;
     background-color: #0e88eb;
     color: #ffffff;   
}

QPushButton:enabled:hover:pressed{
    font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #0e88eb;
     border-radius: 2px;
     background-color: #0d80de;
     color: #ffffff;   
}</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_bSave">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!enabled{
     font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #e6e6e6;
     border-radius: 2px;
     background-color: #ffffff;
     color: #e6e6e6;   
}
QPushButton:enabled:!hover{
     font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #0e88eb;
     border-radius: 2px;
     background-color: #ffffff;
     color: #0e88eb;   
}

QPushButton:enabled:hover{
    font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #0e88eb;
     border-radius: 2px;
     background-color: #0e88eb;
     color: #ffffff;   
}

QPushButton:enabled:hover:pressed{
    font-family: 微软雅黑;
     font-size: 14px;
     border: 1px solid #0e88eb;
     border-radius: 2px;
     background-color: #0d80de;
     color: #ffffff;   
}</string>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZegoImageButton</class>
   <extends>QPushButton</extends>
   <header location="global">ZegoImageButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
